Branch-SDK/src/main/java/io/branch/referral/ServerRequestInitSession.java

```diff
@@ -168,6 +168,17 @@ void updateLinkReferrerParams() {
             } catch (JSONException ignore) {
             }
         }
+
+        String appStore = prefHelper_.getAppStoreSource();
+        if(!TextUtils.isEmpty(appStore) && !PrefHelper.NO_STRING_VALUE.equals(appStore)){
```

Comment:
TODO: Replace the two conditions with the collapsed `isNullOrEmptyOrBlank` over in setAppStore